horovod/spark/lightning/remote.py

```diff
@@ -207,7 +210,7 @@ def on_epoch_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -
             print("Creating trainer with: \n ", kwargs)
             trainer = Trainer(**kwargs)
 
-            if trainer.profiler:
+            if profiler != 'simple' and trainer.profiler:
```

Comment:
why `simple_profiler` doesn't need to set `dirpath` here?

Reply:
it will be easier to just print the simple profiler log to stdout. It is not big, no need to put in file.